3.10/aql/functions-date.md

```diff
@@ -224,6 +224,25 @@ Return the weekday number of *date*.
   - 5 – Friday
   - 6 – Saturday
 
+**Examples**
+
+{% aqlexample examplevar="examplevar" type="type" query="query" bind="bind" result="result" %}
+  @startDocuBlockInline datedyofwk1
+	@EXAMPLE_AQL{datedyofwk1}
```

Comment:
My only concern is with indentation. Not sure if it matters for Docublocks but the [Readme ](https://github.com/arangodb/docs#adding-a-new-aql-example)shows keywords left aligned and states 'preferred' indentation.

Reply:
Looks like it doesn't matter, but for consistency and readability purposes, i'd still align pair elements, i.e. startDocuBlock/endDocuBlock and EXAMPLE_AQL/END_EXAMPLE_AQL